cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -924,24 +926,27 @@ if(fftw3)
   endif()
 endif()
 if(builtin_fftw3)
-  set(FFTW_VERSION 3.3.8)
-  message(STATUS "Downloading and building FFTW version ${FFTW_VERSION}")
-  set(FFTW_LIBRARIES ${CMAKE_BINARY_DIR}/lib/libfftw3.a)
-  ExternalProject_Add(
-    FFTW3
-    URL ${lcgpackages}/fftw-${FFTW_VERSION}.tar.gz
-    URL_HASH SHA256=6113262f6e92c5bd474f2875fa1b01054c4ad5040f6b0da7c03c98821d9ae303
-    INSTALL_DIR ${CMAKE_BINARY_DIR}
-    CONFIGURE_COMMAND ./configure --prefix=<INSTALL_DIR>
-    BUILD_COMMAND make CFLAGS=-fPIC
-    LOG_DOWNLOAD 1 LOG_CONFIGURE 1 LOG_BUILD 1 LOG_INSTALL 1 LOG_OUTPUT_ON_FAILURE 1
-    BUILD_IN_SOURCE 1
-    BUILD_BYPRODUCTS ${FFTW_LIBRARIES}
-    TIMEOUT 600
-  )
-  set(FFTW_INCLUDE_DIR ${CMAKE_BINARY_DIR}/include)
-  set(FFTW3_TARGET FFTW3)
-  set(fftw3 ON CACHE BOOL "Enabled because builtin_fftw3 requested (${fftw3_description})" FORCE)
+  if(NOT fftw3)
+    message(WARNING "fftw3 is OFF, hence ignoring the activated setting 'builtin_fftw3'")
+  else()
```

Comment:
This does not feel quite right.  This seems to now require *both*
```
-Dbuiltin_fftw3=ON -Dffw3=ON
```
to enable `ffw3` with builtin or is some other part of the `CMakeList` allowing the simpler syntax?
I.e. like https://github.com/root-project/root/pull/18413/files#r2044449280 maybe we 
need to move old line 944 here-ish, remove the `FORCE` and test afterwards?